lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -394,32 +399,45 @@ def _lazy_headless():
     import os
     import sys
 
+    backend, deps = sys.argv[1:]
+    deps = deps.split(',')
+
     # make it look headless
     os.environ.pop('DISPLAY', None)
     os.environ.pop('WAYLAND_DISPLAY', None)
 
     # we should fast-track to Agg
     import matplotlib.pyplot as plt
     plt.get_backend() == 'agg'
-    assert 'PyQt5' not in sys.modules
+    for dep in deps:
+        assert dep not in sys.modules
 
-    # make sure we really have pyqt installed
-    import PyQt5  # noqa
-    assert 'PyQt5' in sys.modules
+    # make sure we really have dependencies installed
+    for dep in deps:
+        importlib.import_module(dep)
+        assert dep not in sys.modules
 
     # try to switch and make sure we fail with ImportError
     try:
-        plt.switch_backend('qt5agg')
+        plt.switch_backend(backend)
     except ImportError:
         ...
     else:
         sys.exit(1)
 
 
 @pytest.mark.skipif(sys.platform != "linux", reason="this a linux-only test")
-@pytest.mark.backend('Qt5Agg', skip_on_importerror=True)
-def test_lazy_linux_headless():
-    proc = _run_helper(_lazy_headless, timeout=_test_timeout, MPLBACKEND="")
+@pytest.mark.parametrize("env", _get_testable_interactive_backends())
+def test_lazy_linux_headless(env):
+    tests = []
```

Comment:
This is always empty? The test does nothing...

Reply:
🤦🏻 and it is broken in other ways.  No-op tests do reliably pass!